### PR TITLE
[EGD-7257] Czech localization (by @lukascalda)

### PR DESCRIPTION
**Pull request by GravisZro**
This is resubmission of Czech localization from PR #2019 which was prematurely closed before it was merged.

Issue #1990 can be closed if this PR is merged.

### CODE REVIEW

**Comment by SP2FET** [comment]
@lukascalda :)

**Comment by lukascalda** [comment]
Hi guys, I got lost a bit here. Should I take some actions?

**Comment by GravisZro** [comment]
> Hi guys, I got lost a bit here. Should I take some actions?

@lukascalda, you're here!

There are actually several of new strings that need translating.  I'm not sure if they existed before but there are "profile" files that also need adding.  The profiles seem to be for key entry and a very short.

I've added a script ([`missing_strings.sh`](https://github.com/GravisZro/MuditaOS/blob/localization/czech/missing_strings.sh)) to check what isn't translated.  If you want to avoid all the confusing git rebase stuff then you can just [fork my repo](https://github.com/GravisZro/MuditaOS/tree/localization/czech), edit the files, do a PR to _my_ repo, and I'll just update the files here.

Do note that I've normalized the file names to "Chesky" to match the other normalized translation file names.  git doesn't seem to handle unicode file names very well.

**Comment by lukascalda** [comment]
Okay, I get it. Made some mistakes before. Gotta learn github better. I have made some slight tweaks to my translation, still some lines not translated (190-228) - app related, device should be ready.
It was not merged before, because there was missing review. So what now? New pull request?

**Comment by GravisZro** [comment]
@lukascalda,
The reason your PR wasn't merged due to multiple issues:
* commits need to be collapsed into a single commit
* [commit message was non-conformant](https://github.com/mudita/MuditaOS/blob/master/doc/development_workflow.md#commit-changes)
* needed two reviews
* need to rebase it so it can be merged
* do not close the PR

You can fix all these issues, make a PR and keep it up to date until it gets merged. 

Alternatively, you can...
* [clone my repo](https://github.com/GravisZro/MuditaOS) (`git clone git@github.com:GravisZro/MuditaOS.git`
* switch to the `localization/czech` branch (`git checkout localization/czech`)
* run `missing_strings.sh` script to identify which strings still need translating
* translate the missing strings
* post the translated strings [in a gist](https://gist.github.com/lukascalda)

and I'll make sure it all gets merged properly.

File: missing_strings.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
```
Comment (pholat) [inline missing_strings.sh:1]: this script has little sense, it's fairly easy to check against english.json with few lines of python which have native JSON support. Please leave only the translation. 
Also, kdb files are not needed, so remove them?

**Pull request closed without merging**